add otel in focus for june (#2963)
open-telemetry/opentelemetry.io: content/en/blog/2023/otel-in-focus-06.md

  InetSocketAddressNetClientAttributesGetter, and introduction of new HTTP and
  network semantic conventions.
- New javaagent instrumentation for Quarkus RESTEasy Reactive and Reactor Kafka.
- Enhancements including improvements in Micrometer bridge, Ktor
  instrumentations, AWS SDK support, OkHttp 3, Jetty 11, Spring Boot, AWS Lambda
  tracing, and type matching.
- Bug fixes related to logback MDC instrumentation, kafka metrics reporter,
  jetty context leak, filtering scalar Mono/Flux instances, and others.

#### [Javascript](/docs/instrumentation/js/)

[Version 1.14](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.14.0)
includes an optional `forceFlush` method to the `SpanExporter` interface.

#### [.NET](/docs/instrumentation/net/)

[Version 1.5.1](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.5.1)
has been released, as well as 1.5.0. 1.5.1 is a fix for issues introduced in
1.5.0, and 1.5.0 consisted of the following major changes:

- The bug introduced in 1.5.0-rc.1 that caused the "Build" extension to return
  null has been fixed.
- The Exemplars feature has been marked as internal and will be added back in

>   the 1.6.\* prerelease versions.

Reviewer: What's the `\*` for? Were you planning to add a footnote?

Author: Looks like some escaping gone wild. Will fix.

- A new overload has been added for configuring MeterProviderBuilders while the
  IServiceCollection is modifiable.
- The Console and OpenTelemetryProtocol exporters no longer support exporting
  Exemplars.
- The ASP.NET Core instrumentation has been updated to fix an issue where
  baggage was cleared when the Activity stopped and to add a direct reference to
  System.Text.Encodings.Web.

> - The Http instrumentation has been updated to fix an issue with missing metric

Reviewer: ```suggestion
- The HTTP instrumentation has been updated to fix an issue with missing metric
```

  data in network failures and to improve performance by avoiding boxing of
  common status code values.

## Project and Community Updates

### OTLP 1.0 is here!

> OTLP 1.0 has (or will be soon) released! We'll publish more details about what

Reviewer: ```suggestion
OTLP 1.0 is out (or will soon be)! We'll publish more details about what
```

this means on the blog at some later point. Keep in mind that just because the
specification and protocol are 1.0, this does not mean all OpenTelemetry SIGs
have adopted or released it. Please keep an eye out for future information.

This is still a major accomplishment, and a huge congratulations is in order to
everyone who has contributed. Thank you!

### From the blog...

[End-User Q&A: OTel at Farfetch](/blog/2023/end-user-q-and-a-03/) is the latest
in the End-User Q&A series, featuring Iris Dyrmishi of Farfetch.

[Creating a Kubernetes Cluster with Runtime Observability](/blog/2023/k8s-runtime-observability/)
discusses how to use recent tracing features in Kubernetes components to better

> understand your k8s clusters.

Reviewer: Make these into a list?
